SPECIFICATION.md

```diff
@@ -200,3 +200,72 @@ Whenever a new feature is added to the SDK that does not induce a breaking chang
 ## Patch Versions
 
 Whenever a new release is warranted to only preform bug fixes it will be considered a patch version.
+
+# Git Structure
+
+As the APIs evolve over time, there will be instances were multiple versions of the SDKs will need to be supported at the same time. To better facilitate this, the Server SDK repositories will follow a common version branching structure for releases. This structure is similiar to, but not exactly, [Gitflow](https://nvie.com/posts/a-successful-git-branching-model/).
```

Comment:
Does this mean we'll be adopting the push to develop branch and then merge develop to master as well?

Reply:
Nevermind, I see it explained below 😛